client/src/main/python/slipstream/NodeDecorator.py

```diff
@@ -70,6 +70,7 @@ class NodeDecorator(object):
     RUN_BUILD_RECIPES_KEY = 'run-build-recipes'
     PLATFORM_KEY = 'platform'
     LOGIN_USER_KEY = 'loginUser'
+    LOGIN_PASS_KEY = 'login.password'
```

Comment:
I would like but it will generate a lot of side effects.
So I would wait for the new persistence schema before changing it.
I created the following task slipstream/SlipStream#78